ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java

```diff
@@ -54,7 +54,7 @@
 @ToolAnnotation(RAGTool.TYPE)
 public class RAGTool extends AbstractRetrieverTool {
     public static final String TYPE = "RAGTool";
-    private static String DEFAULT_DESCRIPTION = "Use this tool to run any model.";
+    private static String DEFAULT_DESCRIPTION = "Use this tool to retrieve helpful information to answer question.";
```

Comment:
retrieve helpful information `from where`?

Reply:
This tool is different with VectorDBTool which is to retrieve helpful information. This RAGTool is to help answer questions with user configured knowledge base which provide helpful information.

Reply:
Based on the defination of [RAG](https://aws.amazon.com/what-is/retrieval-augmented-generation/#:~:text=Augmented%20Generation%20requirements%3F-,What%20is%20Retrieval%2DAugmented%20Generation%3F,sources%20before%20generating%20a%20response.), I will suggest we could describe RAG tool as to `optimize the output of a large language model to answer questions`.

Reply:
Agreed. Will change the description.